sharktank/tests/models/llama/kv_cache_test.py

```diff
@@ -0,0 +1,245 @@
+import unittest
+import torch
+import torch.nn as nn
+from sharktank.models.llama.llama import (
+    PagedLlamaAttentionBlock,
+    PagedKVCache,
+    DirectKVCache,
+)
+from sharktank.models.llama.testing import *
+from sharktank.layers.rotary_embedding import RotaryEmbeddingLayer
+from ...layers import *
+
+
+class KVCacheTest(unittest.TestCase):
+    def setUp(self):
+        self.block_count = 1
+        self.seq_len = 16
+        self.head_count = 32
+        self.head_dim = 128
+        self.ffn_dim = 11008
+        self.head_count_kv = 32
+        self.block_seq_stride = 16
+        self.rms_epsilon = 1e-5
+        self.rope_dimension_count = 128
+        self.max_seq_len = 4096
+        self.start_positions = torch.tensor([8])
+        self.bs = 1
```

Comment:
We still have a TODO to implement batch size 1 in the llama model I think: https://github.com/nod-ai/sharktank/issues/40

I wonder if it would help to run this test parameterized (https://stackoverflow.com/a/34094) over `bs == 1`, `bs == 2`, and `bs == 4`?

Reply:
I see, I'm not sure how that would work yet. I can take a look at that later, just want to get the fix and a couple of basic tests in for now.

Reply:
I think its ok to get in as is for now, but file an issue for yourself so you can circle back to it later.